TheWarWithin/MageArcane.lua

```diff
@@ -761,25 +761,112 @@ end )
 local totm_casts = 0
 local clearcasting_consumed = 0
 
+-- Intuition Bad Luck Protection tracking
```

Comment:
Are we consistently using camelCaseNaming for "real" variables vs. using_underscores for variables we modify inside the virtual state?